contrib/monitoring/fdb_c_version.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+#
+# fdb_c_version.py
+#
+#  This source file is part of the FoundationDB open source project
+#
+#  Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argparse
+import ctypes
+import sys
+import platform
+import os
+
+def error(message):
+    print(message)
+    sys.exit(1)
+
+def get_version_string(library_path):
+    try:
+        lib = ctypes.cdll.LoadLibrary(os.path.abspath(library_path))
+    except Exception as e:
+        error('Could not load library %r: %s' % (library_path, e.message))
```

Comment:
I got error at this line on Mac:
```
Traceback (most recent call last):
  File "/Users/jzhou/src/foundationdb/../fdb_c_version.py", line 87, in <module>
    print(get_version_string(args.library_path))
  File "/Users/jzhou/src/foundationdb/../fdb_c_version.py", line 35, in get_version_string
    error('Could not load library %r: %s' % (library_path, e.message))
AttributeError: 'OSError' object has no attribute **'message'**
```

Reply:
Do you have a libfdb_c.dylib installed somewhere that would be picked up on the default search path? If not, you should also be able to give it a path to a file.

Reply:
I am suggesting we should fix `e.message` here.

Reply:
Sure, I can look at that too in my subsequent cleanup.